app/commands.py

```diff
@@ -611,10 +616,31 @@ async def requests(ctx: Context) -> Optional[str]:
             l.append(f"Failed to find requested map ({map_id})?")
             continue
 
-        l.append(f"[{p.embed} @ {dt:%b %d %I:%M%p}] {bmap.embed}.")
+        l.append(f"#{id}: {bmap.embed}\n({dt:%b %d %I:%M%p}) Requested for "
+                 + f"{requested_status} by {p.embed}. "
+                 + f"{'No additional comment was specified.' if comment == '' else f'Additional comment: {comment}'}")
 
     return "\n".join(l)
 
+@command(Privileges.NOMINATOR, aliases=["dr"], hidden=True)
+async def declinerequest(ctx: Context) -> Optional[str]:
+    """Removes a map request by it's ID."""
+    if len(ctx.args) != 1:
+        return "Invalid syntax: !declinerequest <ID>"
+
+    if await app.state.services.database.fetch_one(
+        "SELECT 1 FROM map_requests WHERE active = 1 AND id = :id",
+        {"id": ctx.args[0]}
+    ):
+        return f"No active map request with ID #{ctx.args[0]} found."
```

Comment:
is this missing an `is None` check? perhaps i'm misreading

Reply:
Got this if check from somewhere else, isn't if ... and if ... is not None the same?